module.exports = {

"[project]/.next-internal/server/app/auth/login/page/actions.js [app-rsc] (server actions loader, ecmascript)": ((__turbopack_context__) => {

var { m: module, e: exports } = __turbopack_context__;
{
}}),
"[project]/src/app/favicon.ico.mjs { IMAGE => \"[project]/src/app/favicon.ico (static in ecmascript)\" } [app-rsc] (structured image object, ecmascript, Next.js Server Component)": ((__turbopack_context__) => {

__turbopack_context__.n(__turbopack_context__.i("[project]/src/app/favicon.ico.mjs { IMAGE => \"[project]/src/app/favicon.ico (static in ecmascript)\" } [app-rsc] (structured image object, ecmascript)"));
}),
"[project]/src/app/layout.tsx [app-rsc] (ecmascript, Next.js Server Component)": ((__turbopack_context__) => {

__turbopack_context__.n(__turbopack_context__.i("[project]/src/app/layout.tsx [app-rsc] (ecmascript)"));
}),
"[project]/src/app/auth/login/page.tsx [app-rsc] (ecmascript)": ((__turbopack_context__) => {

var { m: module, e: exports } = __turbopack_context__;
{
// En tu archivo login.tsx
const handleSubmit = async (e)=>{
    e.preventDefault(); // Previene la recarga de la página
    setError('');
    console.log('1. El formulario fue enviado.');
    try {
        console.log('2. Intentando llamar al servicio de login...');
        await login({
            email,
            password
        });
        // Si llegamos aquí, el login en el backend fue exitoso.
        console.log('3. Login exitoso. Redirigiendo a /inicio...');
        router.push('/inicio');
    } catch (err) {
        // Si algo sale mal en la llamada a `login()`, se ejecutará esto.
        console.error('4. Ocurrió un error:', err);
        setError('Credenciales inválidas o error en el servidor.');
    }
};
}}),
"[project]/src/app/auth/login/page.tsx [app-rsc] (ecmascript, Next.js Server Component)": ((__turbopack_context__) => {

__turbopack_context__.n(__turbopack_context__.i("[project]/src/app/auth/login/page.tsx [app-rsc] (ecmascript)"));
}),
"[externals]/next/dist/shared/lib/no-fallback-error.external.js [external] (next/dist/shared/lib/no-fallback-error.external.js, cjs)": ((__turbopack_context__) => {

var { m: module, e: exports } = __turbopack_context__;
{
const mod = __turbopack_context__.x("next/dist/shared/lib/no-fallback-error.external.js", () => require("next/dist/shared/lib/no-fallback-error.external.js"));

module.exports = mod;
}}),

};

//# sourceMappingURL=%5Broot-of-the-server%5D__59013932._.js.map